andela/Ah-backend-aquaman
# #164069232 Add tags to articles

#### What does this PR do?
Adds a feature that enables authenticated authors to tag their articles
#### Description of Task to be completed?
add tagList to Article model

#### How should this be manually tested?
* Clone the repo, cd into Ah-backend-aquaman and checkout the branch ft-create-tags-164069232 
* pip install -r requirements.txt
* python manage.py migrate
* python manage.py runserver
* create an account, verify your email, login and create articles with alist of tags
* use postman to test the tags endpoint /api/tags/
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#164069232](https://www.pivotaltracker.com/story/show/164069232)
#### Screenshots (if appropriate) 

`post an article with tags`
<img width="1126" alt="Screenshot 2019-03-14 at 15 42 13" src="https://user-images.githubusercontent.com/20795487/54358081-91541900-4670-11e9-818b-4ada1bf5f458.png">

`View all tags`
<img width="1127" alt="Screenshot 2019-03-14 at 15 42 33" src="https://user-images.githubusercontent.com/20795487/54358109-a03acb80-4670-11e9-9021-b142ae978234.png">
